lib/cli/src/commands/app/secrets/create.rs

```diff
@@ -0,0 +1,271 @@
+            Ok(app.id.into_inner())
+        }
+    }
+
+    /// Given a list of secrets, checks if the given secrets already exist for the given app and
+    /// returns a list of secrets that must be upserted.
+    async fn filter_secrets(
+        &self,
+        client: &WasmerClient,
+        app_id: &str,
+        secrets: Vec<Secret>,
+    ) -> anyhow::Result<Vec<Secret>> {
+        let names = secrets.iter().map(|s| &s.name);
+        let app_secrets =
+            wasmer_api::query::get_all_app_secrets_filtered(client, app_id, names).await?;
+        let mut sset = HashSet::<String>::from_iter(app_secrets.iter().map(|s| s.name.clone()));
+        let mut ret = HashMap::new();
+
+        for secret in secrets {
+            if sset.contains(&secret.name) {
+                if self.non_interactive {
+                    anyhow::bail!("Cannot create secret '{}' as it already exists. Use the `update` command instead.", secret.name.bold());
+                } else {
+                    if ret.contains_key(&secret.name) {
+                        eprintln!(
+                            "Secret '{}' appears twice in the input file.",
+                            secret.name.bold()
+                        );
+                    } else {
+                        eprintln!(
+                            "Secret '{}' already exists for the selected app.",
+                            secret.name.bold()
+                        );
+                    }
+                    let theme = ColorfulTheme::default();
+                    let res = dialoguer::Confirm::with_theme(&theme)
+                        .with_prompt("Do you want to update it?")
+                        .interact()?;
+
+                    if !res {
+                        eprintln!("Cannot create secret '{}' as it already exists. Use the `update` command instead.", secret.name.bold());
+                    }
+                }
+            }
+
+            sset.insert(secret.name.clone());
+            ret.insert(secret.name, secret.value);
+        }
+
+        Ok(ret
+            .into_iter()
+            .map(|(name, value)| Secret { name, value })
+            .collect())
+    }
+
+    async fn create(
+        &self,
+        client: &WasmerClient,
+        app_id: &str,
+        secrets: Vec<Secret>,
+    ) -> Result<(), anyhow::Error> {
+        let res = wasmer_api::query::upsert_app_secrets(
+            client,
+            app_id,
+            secrets.iter().map(|s| (s.name.as_str(), s.value.as_str())),
+        )
+        .await?;
+        let res = res.context(
+            "Backend did not return any payload to confirm the successful creation of the secret!",
+        )?;
+
+        if !res.success {
+            anyhow::bail!("Secret creation failed!")
+        } else {
+            if !self.quiet {
+                eprintln!("Succesfully created secret(s):");
+                for secret in secrets {
+                    eprintln!("{}", secret.name.bold());
```

Comment:
Should show a hint that secrets are not deployed yet, and to re-deploy the app to apply the changes.

Reply:
Added a message after printing that secrets were created/updated: 
```
Info: In order for secrets to appear in your app, re-deploy it.
```